gtk3-no-gi/sample/misc/linkbutton.rb

```diff
@@ -8,12 +8,12 @@
   $Id: linkbutton.rb,v 1.1 2006/10/21 16:58:00 mutoh Exp $
 =end
 
-require 'gtk3'
+require "gtk3"
 
 window = Gtk::Window.new("Gtk::LinkButton sample")
 window.signal_connect("destroy"){Gtk.main_quit}
 
-vbox = Gtk::VBox.new
+vbox = Gtk::Box.new(:vertical,0)
```

Comment:
`spacing` can be omitted.

```suggestion
vbox = Gtk::Box.new(:vertical)
```